### ISSUE TITLE
Broken recipe: strategywiki

### ISSUE THREAD

**Issue by circumnaviganto** [openzim/zim-requests#685]
Hey,

Noticed that the recipe `strategywiki` has failed all 8 times it has been run.

I checked the log for the latest run of this, and it appears that it's misconfigured.
It returns the error "Request failed with status code 404", which if you check line 1833 of the logfile for this run, appears to be triggered by a GET request at URL `https://strategywiki.org/wiki/Main_Page/w/api.php?action=query&meta=siteinfo&format=json&siprop=general|namespaces|statistics|variables|category|wikidesc`, which seems illogical, as the API for this wiki is not under `/wiki/Main_Page/`.
Requesting `https://strategywiki.org/w/api.php?action=query&meta=siteinfo&format=json&siprop=general|namespaces|statistics|variables|category|wikidesc` instead returns a much more reasonable response.

As I'm not involved with the project, I am unable to rectify this issue, but is it possible that someone associated with the project can make the required correction?

Thanks.

**Comment by Popolechien** [comment]
@circumnaviganto Thanks for this, we'll look into it later this week and see what gives.

**Comment by RavanJAltaie** [comment]
This is related to mwoffliner expected update that's supposed to fix the scraper issues.
It's expected to be don by end of this year, once the scraper is updated we will review all mwoffliner recipes.

**Comment by kelson42** [comment]
@RavanJAltaie No, recipe is wrong. This is pretty obvious, look at the error message. If you think this is a problem with the software please open a ticket in MWoffliner repository or/and link it from here.

**Comment by kelson42** [comment]
@Popolechien comment on wrong ticket?

**Comment by Popolechien** [comment]
welp yes. copying to [Kiwix-android#2561](https://github.com/kiwix/kiwix-android/issues/2561)

**Comment by RavanJAltaie** [comment]
@kelson42 I reviewed the error and the recipe and couldn't understand the error:

"Unable to connect to S3, either S3 login credentials are wrong or bucket cannot be found
                            Bucket used: org-kiwix-mwoffliner
                            End point used: https://s3.us-west-1.wasabisys.com/
                            Public IP used: 65.28.105.18"
             
What does that mean?

**Comment by kelson42** [comment]
@RavanJAltaie This an operational problem which had been fixed yesterday. just restart the recipe and it should be ok

**Comment by RavanJAltaie** [comment]
Recipe is still failing with error: Request failed with status code 404 
As discussed, I'll tag the issue as upstream

**Comment by benoit74** [comment]
I have disabled the recipe for now.

**Comment by benoit74** [comment]
I fixed the recipe and restarted it, to be checked sooner or later: https://farm.openzim.org/recipes/strategywiki/

**Comment by benoit74** [comment]
ZIM is rather incomplete, many pages are missing when there is a forward slash, linked to https://github.com/openzim/mwoffliner/issues/2105 and/or https://github.com/openzim/mwoffliner/issues/726 obviously.

**Comment by Popolechien** [comment]
Just to document the slash issue, see e.g. at https://dev.library.kiwix.org/viewer#strategywiki_en_all_maxi_2025-03/A/Battlestations%3A_Midway where clicking on `a new challenge mission` fails to open https://dev.library.kiwix.org/viewer#strategywiki_en_all_maxi_2025-03/A/Battlestations%3A_Midway/Raid_on_Truk

**Comment by Popolechien** [comment]
Relaunched with latest MWoffliner release. Task https://farm.openzim.org/pipeline/025245a4-e56e-41c4-899b-40adbf5108d4

**Comment by benoit74** [comment]
Task failed at the very end but behavior was a bit weird, so I've restarted it with `dev` image to check we do not have an unfixed bug.

**Comment by Popolechien** [comment]
[Failed again](https://farm.openzim.org/recipes/strategywiki_en_all)

**Comment by benoit74** [comment]
Looks like scraper is now blacklisted ... Nothing we can fix on our side ATM
